imap_processing/hi/l1a/science_direct_event.py

```diff
@@ -245,86 +252,60 @@ def create_dataset(de_data_list: list, packet_met_time: list) -> xr.Dataset:
             metaevent_time_in_ns += half_tick_ns
             continue
 
+        data_dict["meta_event_met"].append(metaevent_time_in_ns)
         # calculate direct event time using time information from metaevent
         # and de_tag. epoch in this dataset uses this time of the event
-        de_time_in_ns = (
+        de_met_in_ns = (
             metaevent_time_in_ns
             + event["de_tag"] * LOOKED_UP_DURATION_OF_TICK * MICROSECOND_TO_NS
         )
-        de_time = get_direct_event_time(de_time_in_ns)
-        de_met_time.append(de_time)
-        esa_step.append(current_esa_step)
+        data_dict["event_met"].append(de_met_in_ns)
+        data_dict["epoch"].append(get_direct_event_time(de_met_in_ns))
+        data_dict["esa_stepping_num"].append(current_esa_step)
         # start_bitmask_data is 1, 2, 3 for detector A, B, C
         # respectively. This is used to identify which detector
         # was hit first for this current direct event.
-        trigger_id.append(event["start_bitmask_data"])
-        tof_1.append(event["tof_1"])
-        tof_2.append(event["tof_2"])
-        tof_3.append(event["tof_3"])
+        data_dict["trigger_id"].append(event["start_bitmask_data"])
+        data_dict["tof_1"].append(event["tof_1"])
+        data_dict["tof_2"].append(event["tof_2"])
+        data_dict["tof_3"].append(event["tof_3"])
         # IMAP-Hi like to keep de_tag value for diagnostic purposes
-        de_tag.append(event["de_tag"])
+        data_dict["de_tag"].append(event["de_tag"])
         # add packet time to ccsds_met list
-        ccsds_met.append(packet_met_time[index])
+        data_dict["ccsds_met"].append(packet_met_time[index])
+
+    # Load the CDF attributes
+    cdf_manager = CdfAttributeManager(imap_module_directory / "cdf" / "config")
+    cdf_manager.load_global_attributes("imap_hi_global_cdf_attrs.yaml")
+    cdf_manager.load_variable_attributes("imap_hi_variable_attrs.yaml")
+
+    # Inject sensor head into Logical_source
+    gattrs = cdf_manager.get_global_attributes("imap_hi_l1a_de_attrs")
```

Comment:
can we spell out global?
```suggestion
    de_global_attrs = cdf_manager.get_global_attributes("imap_hi_l1a_de_attrs")
```